src/main/java/nl/rijksoverheid/mev/common/util/BSNValidator.java

```diff
@@ -14,10 +14,34 @@ public final class BSNValidator {
     private static final int MIN_BSN_LEN = 8;
     private static final int ELF_PROEF_DENOMINATOR = 11;
 
+    /**
+     * Valideer één of meerdere bsns
+     *
+     * @param bsns de lijst met bsns
+     * @return of de bsns valide zijn
+     * @throws GezagException wanneer één van de bsns niet voldoet aan de elf
+     * proef
+     */
+    public boolean isValid(final List<String> bsns) {
```

Comment:
In general prefer `Collection<String>` over `List<String>` in arguments, so the method can be called with any collection.